LaunchAtLogin/LaunchAtLogin.swift

```diff
@@ -1,21 +1,46 @@
+import Combine
 import Foundation
 import ServiceManagement
 
-public struct LaunchAtLogin {
-	private static let id = "\(Bundle.main.bundleIdentifier!)-LaunchAtLoginHelper"
+public class LaunchAtLogin: NSObject {
+    public static let shared = LaunchAtLogin()
 
-	public static var isEnabled: Bool {
-		get {
-			guard let jobs = (SMCopyAllJobDictionaries(kSMDomainUserLaunchd).takeRetainedValue() as? [[String: AnyObject]]) else {
-				return false
-			}
+    public static var isEnabled: Bool {
+        get { shared.isEnabled }
+        set { shared.isEnabled = newValue }
+    }
 
-			let job = jobs.first { $0["Label"] as! String == id }
+    @available(OSX 10.15, *)
+    public static var publisher: AnyPublisher<Bool, Never> = {
```

Comment:
I think this should be a CurrentValueSubject, so the initial value is also available directly on the publisher.